avm/res/container-instance/container-group/main.bicep

```
@@ -21,43 +21,20 @@ param osType string = 'Linux'
 ])
 param restartPolicy string = 'Always'
 
-@description('Optional. Specifies if the IP is exposed to the public internet or private VNET. - Public or Private.')
-@allowed([
-  'Public'
-  'Private'
-])
-param ipAddressType string = 'Public'
-
 @description('Optional. The image registry credentials by which the container group is created from.')
 param imageRegistryCredentials imageRegistryCredentialType[]?
 
 @description('Optional. Location for all Resources.')
 param location string = resourceGroup().location
 
-@description('Optional. Specify level of protection of the domain name label.')
-@allowed([
-  'Noreuse'
-  'ResourceGroupReuse'
-  'SubscriptionReuse'
-  'TenantReuse'
-  'Unsecure'
-])
-param autoGeneratedDomainNameLabelScope string = 'TenantReuse'
-
-@description('Optional. The Dns name label for the resource.')
-param dnsNameLabel string?
-
-@description('Optional. List of dns servers used by the containers for lookups.')
-param dnsNameServers string[]?
-
-@description('Optional. DNS search domain which will be appended to each DNS lookup.')
-param dnsSearchDomains string?
+@description('Optional. The DNS config information for a container group.')
+param dnsConfig dnsConfigType?
 
 @description('Optional. A list of container definitions which will be executed before the application container starts.')
 param initContainers array?
 
-@description('Optional. Resource ID of the subnet. Only specify when ipAddressType is Private.')
-param subnetResourceId string?
+@description('Optional. The subnet resource IDs for a container group.')
+param subnetIds containerGroupSubnetIdType[]?
```

```suggestion
@description('Optional. The subnets to use by the container group.')
param subnets containerGroupSubnetIdType[]?
```
Might be better to use a more 'generic' name as it's not only the resourceIds now, but also optionally the displayName.

---

I have renamed the param